adam-core/src/main/scala/org/bdgenomics/adam/converters/VariantAnnotations.scala

```diff
@@ -0,0 +1,140 @@
+/**
+ * Licensed to Big Data Genomics (BDG) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The BDG licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.bdgenomics.adam.converters
+
+import htsjdk.variant.vcf.VCFConstants
+import htsjdk.variant.variantcontext.VariantContext
+import org.bdgenomics.formats.avro.{ TranscriptEffect, Variant, VariantAnnotation, VariantAnnotationMessage }
+import org.bdgenomics.formats.avro.VariantAnnotationMessage._
+import org.bdgenomics.utils.misc.Logging
+import scala.collection.JavaConverters._
+
+object VariantAnnotations extends Serializable with Logging {
```

Comment:
In #1078 the code in this class will be merged with the already existing `VariantAnnotationConverter`.  Would you still want that change here?